docker/README.md

```diff
@@ -7,6 +7,18 @@ This Docker image is designed to:
 * be spun up in front of Trino clusters by mounting in a configuration file
 * simplify deployment into an orchestration system
 
+## Production Setup
+
+Current healthcheck configurations for docker compose files is for develop purpose.  
+So we advise changing it to settings as following: 
+
+|  configuration  | [gateway-compose.yml](./gateway-compose.yml) | [postgres-backend-compose.yml](./postgres-backend-compose.yml) |
+|:---------------:|:--------------------------------------------:|:--------------------------------------------------------------:|
+|   interval(s)   |                      10                      |                               10                               |
+|   timeout(s)    |                      3                       |                               3                                |
+|     retries     |                      3                       |                               3                                |
+| start_period(s) |                      10                      |                               10                               |
+
```

Comment:
added production setup here PTAL @mosabua

Reply:
The basis for these numbers are unclear to me. I believe this config should be determined based on the workload. I don't think mentioning these numbers is a good idea. 

If this config fits all environments, we should change the yml files instead of docs.

Reply:
Well as @rdsarvar [saids](https://github.com/trinodb/trino-gateway/pull/238/files#r1481547210), current config is for internal test purpose and not really suitable for production.
I agree to the point that the numbers depend based on the workload, then how about just simply mentioning that current setup is for develop purpose and user's should change it when using in production?